### PR TITLE
continue work on server and db; have not completed db

### PR DESCRIPTION
**Pull request by lugyin**

### CODE REVIEW

File: server/db/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+var Sequelize = require("sequelize");
+var sequelize = new Sequelize("mydb1", "luyin", "",{
+	host: 'localhost',
+	dialect: 'postgres',
+
+	pool: {
+    max: 5,
```
Comment (awatemonosan) [inline server/db/index.js]: Min should come before max

File: server/db/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+var Sequelize = require("sequelize");
```
Comment (awatemonosan) [inline server/db/index.js]: I think that index.js is not a very descriptive name for this file

**Comment by awatemonosan** [comment]
Seems good. Little confused on the name of the db file though. :+1:

**Comment by bcjl** [comment]
Sure :+1:

**Pull request merged**